scripts/get-baseline-versions.js

```diff
@@ -1,6 +1,10 @@
 require = require("@httptoolkit/esm")(module);
 bbm = require("baseline-browser-mapping");
 
+const packageJson = require(process.cwd() + "/package.json");
+const incomingConfig = packageJson["browserslist-config-baseline"] ?? {};
```

Comment:
This will not work if we’re in a subdirectory. We should walk back until we see our config, e.g.

```suggestion
const fs = require("fs");
const path = require("path");

function readConfig(loc, name) {
  do {
    try {
      const pkg = require(path.join(loc, "package.json"));
      if (pkg.hasOwnProperty(name)) return pkg[name];
    } catch {}
  } while (loc !== (loc = path.dirname(loc)));

  return null;
}

const incomingConfig =
  readConfig(process.cwd(), "browserslist-config-baseline") ?? {};
```